trimesh/path/polygons.py

```diff
@@ -885,39 +885,91 @@ def projected(mesh,
     return polygon
 
 
-def second_moment(coords):
+def second_moments(polygon, centered=False):
     """
-    Calculate the second moment of area of a polygon
+    Calculate the second moments of area of a polygon
     from the boundary.
 
     Parameters
     ------------
     coords : (n, 2) float or Polygon
```

Comment:
I forgot to change this line (895, polygons.py) in the docstring: (n,2) does not work anymore. Thank you very much for merging the PR!

Reply:
Fixed, thanks!